mrbelvedereci/templates/layout_full.html

```diff
@@ -14,7 +14,7 @@
 {% block local_header %}
 {% if debug %}
 <div class="slds-notify_container">
-    <div class="slds-notify slds-notify--toast" role="alert">
```

Comment:
Does this fix https://github.com/SalesforceFoundation/mrbelvedereci/issues/96?

Reply:
Yes, there's still an overlay over part of the header but all the header links and the search box are still usable this way.